examples/github/formulas.ts

```diff
@@ -157,84 +180,100 @@ const getUserFormula = makeFormula({
+// to sync, since GitHub's API does not return entities across repos
+// However, a user can set up multiple sync configurations
 // and each one can individually sync from a separate repo.
 // (This is exported so that we can unittest the autocomplete formula.)
 export const repoUrlParameter = makeParameter({
   type: ParameterType.String,
-  name: 'repoUrl',
-  description: 'The URL of the repository to list pull requests from. For example "https://github.com/[org]/[repo]".',
-  // This autocomplete formula will list all of the repos that the current user has access to
-  // and expose them as a searchable dropdown in the UI.
-  // It fetches the GitHub repo objects and then runs a simple text search over the repo name.
+  name: "repoUrl",
+  description:
+    "The URL of the repository to list pull requests from. For example 'https://github.com/[org]/[repo]'.",
+  // This autocomplete formula will list all of the repos that the current
+  // user has access to and expose them as a searchable dropdown in the UI.
+  // It fetches the GitHub repo objects and then runs a simple text search
+  // over the repo name.
   autocomplete: makeMetadataFormula(async (context, search) => {
     let results: GitHubRepo[] = [];
     let continuation: Continuation | undefined;
     do {
-      const response = await getRepos(context, continuation);
+      let response = await getRepos(context, continuation);
       results = results.concat(...response.result);
       ({continuation} = response);
     } while (continuation && continuation.nextUrl);
-    // This helper function can implement most autocomplete use cases. It takes the user's current
-    // search (if any) and an array of arbitrary objects. The final arguments are the property name of
-    // a label field to search over, and finally the property name that should be used as the value
-    // when a user selects a result.
-    // So here, this is saying "search the `name` field of reach result, and use the html_url as the
-    // value once selected".
-    return autocompleteSearchObjects(search, results, 'name', 'html_url');
+    // This helper function can implement most autocomplete use cases. It
+    // takes the user's current search (if any) and an array of arbitrary
+    // objects. The final arguments are the property name of a label field to
+    // search over, and finally the property name that should be used as the
+    // value when a user selects a result.
+    // So here, this is saying "search the `name` field of reach result, and
+    // use the html_url as the value once selected".
+    return autocompleteSearchObjects(search, results, "name", "html_url");
   }),
 });
 
 const baseParameterOptional = makeParameter({
   type: ParameterType.String,
-  name: 'base',
-  description: 'The name of the base branch. For example, "master".',
+  name: "base",
+  description: "The name of the base branch. For example, 'main'.",
```

Comment:
I could not for the life of me figure out how to get VSCode to stop auto-fixing the string. The string we want to be lint-compliant is `"The name of the base branch. For example, \"main\"."` But some combination of prettier and eslint keeps changing it `'The name of the base branch. For example, "main".'` and I couldn't get it to stop doing that. We care about the double-quote around "main" because that's Coda Formula Language syntax for a string literal.

Reply:
FWIW I think it's OK to have the occasional single-quote strings when it means less character escaping.